src/styles/_variables.scss

```diff
@@ -55,6 +55,7 @@ $basic-speed: 150ms;
 // LAYOUT
 $basic-padding: 8px;
 $toolbar-height: 60px;
+$toolbar-secondary-height: 52px;
```

Comment:
Should we rename this to like `dot-edit-toolbar-height` or something edit-mode related?